README.md

```diff
@@ -17,6 +17,10 @@ Download the config (you may need to `brew install awscli` to get the command.
 
 Get Janus credentials for membership.
 
+Create an ssh tunnel to the CODE one-off contributions database:
+1. Clone https://github.com/guardian/contributions-platform
+2. From the contributions-platform project, Run `./contributions-store/contributions-store-bastion/scripts/open_ssh_tunnel.sh -s CODE`
+
```

Comment:
and, if you don't have marauder installed then https://github.com/guardian/prism/tree/master/marauder

Reply:
👍